app/views/students/student_report.pdf.erb

```diff
@@ -124,13 +124,13 @@
                 <span style="font-weight:bold;">
                   <%= format_date(service.date_started) %> -
                   <% if service.discontinued_at.present? %>
-                    <%= "#{format_date(service.try(:discontinued_at))}" %>
+                    <%= "#{format_date(service.discontinued_at)}" %>
                   <% else %>
                     Present
                   <% end %>
                 </span>
               </li>
-              <li><span style="font-style:italic;"><%= service.try(:service_type).try(:name) %></span></li>
+              <li><span style="font-style:italic;"><%= service.service_type.name || "N/A" %></span></li>
```

Comment:
Every service should have a service type, otherwise we wouldn't know what kind of service it is and the database record would be mostly meaningless. And every service type should have a name. So in this case I think removing the `try` statements is a very good move, but we don't need to add "N/A" as a fallback because we should never have a service without a service type or a service type without a name.